.github/new-prs-labeler.yml

```diff
@@ -822,6 +822,23 @@ backend:X86:
   - llvm/lib/TargetParser/X86*
   - llvm/utils/TableGen/X86*
 
+backend:PowerPC:
+  - llvm/include/llvm/IR/IntrinsicsPowerPC.td
+  - llvm/lib/Target/PowerPC/**
+  - llvm/test/CodeGen/PowerPC/**
```

Comment:
For tests, can we use `llvm/test/**/PowerPC/**` and `llvm/test/**/XCOFF/**`?
I assume we are also interested in
```
llvm/test/DebugInfo/Symbolize/XCOFF
llvm/test/DebugInfo/XCOFF
llvm/test/MC/XCOFF
llvm/test/tools/llvm-dwarfdump/XCOFF
llvm/test/tools/llvm-nm/XCOFF
llvm/test/tools/llvm-objcopy/XCOFF
llvm/test/tools/llvm-objdump/XCOFF
llvm/test/tools/llvm-readobj/XCOFF
llvm/test/tools/obj2yaml/XCOFF
llvm/test/tools/yaml2obj/XCOFF
llvm/test/Analysis/CostModel/PowerPC
llvm/test/Analysis/LoopCacheAnalysis/PowerPC
llvm/test/CodeGen/MIR/PowerPC
llvm/test/CodeGen/PowerPC
llvm/test/DebugInfo/PowerPC
llvm/test/ExecutionEngine/RuntimeDyld/PowerPC
llvm/test/Instrumentation/MemorySanitizer/PowerPC
llvm/test/LTO/PowerPC
llvm/test/MC/Disassembler/PowerPC
llvm/test/MC/PowerPC
llvm/test/Transforms/AtomicExpand/PowerPC
llvm/test/Transforms/CodeGenPrepare/PowerPC
llvm/test/Transforms/ConstantHoisting/PowerPC
llvm/test/Transforms/DivRemPairs/PowerPC
llvm/test/Transforms/EarlyCSE/PowerPC
llvm/test/Transforms/GlobalOpt/PowerPC
llvm/test/Transforms/Inline/PowerPC
llvm/test/Transforms/InstCombine/PowerPC
llvm/test/Transforms/LoopDataPrefetch/PowerPC
llvm/test/Transforms/LoopUnroll/PowerPC
llvm/test/Transforms/LoopVectorize/PowerPC
llvm/test/Transforms/SLPVectorizer/PowerPC
llvm/test/Transforms/SimplifyCFG/PowerPC
llvm/test/tools/gold/PowerPC
llvm/test/tools/llvm-exegesis/PowerPC
llvm/test/tools/llvm-objdump/ELF/PowerPC
```

Reply:
Thanks for the lists.

I added more paths related to PowerPC. However I exclude the tools related ones. One cares about the tools can make themselves as the member of the tools related teams. Here we focus on the code generations of the compiler itself.

We always can enlarge this list if we find compiler changes lead to tool tests diff but no hit in the compiler test dir in future.

Reply:
I favor simple `llvm/test/**/PowerPC/**` and `llvm/test/**/XCOFF/**`